### PR TITLE
Add transfer-eth-erc20-from-near to finish the two-directional transfer

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +95,7 @@ node index.js init-near-fun-token
 Now start the services that will relay the information between the chains:
 ```bash
 node index.js start eth-relay
-node index.js start near-relay
+node index.js start near-relay --eth-master-sk 0x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b7501201
```
Comment (ailisp) [inline README.md:98]: I think this should be fine to omit, as eth sk is in config.json?

File: environment/commands/init-near2eth-prover.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            web3.eth.accounts.privateKeyToAccount(RainbowConfig.getParam('eth-master-sk'));
+        web3.eth.accounts.wallet.add(ethMasterAccount);
+        web3.eth.defaultAccount = ethMasterAccount.address;
+        ethMasterAccount = ethMasterAccount.address;
+
+        // Initialize client contract.
+        console.log('Deploying Near2EthClient contract.');
+        const nearProver = new web3.eth.Contract(JSON.parse(
+            // @ts-ignore
+            fs.readFileSync(RainbowConfig.getParam('near2eth-prover-abi-path'))));
```
Comment (ailisp) [inline environment/commands/init-near2eth-prover.js:19]: can also use unblocking version:
```
const fs = require('fs').promises;
await fs.readFile(RainbowConfig.getParam('near2eth-prover-abi-path'))));
```
but i guess readFileSync it's fine

Comment (nearmax) [reply environment/commands/init-near2eth-prover.js:19]: Yes, we are using sync version everywhere in this code..

File: environment/commands/transfer-eth-erc20-from-near.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+                await sleep(sleepSec * 1000);
+            } else {
+                const sleepSec = 10;
+                console.log(`Block ${outcomeBlockHeight.toString()} is not available on the light client yet. Current height of light client is ${clientBlockHeight.toString()}. Sleeping ${sleepSec} seconds.`);
+                await sleep(sleepSec * 1000);
+            }
+        }
+
+        console.log(`Burnt ${JSON.stringify(command.amount)}`);
+        const new_balance = await nearTokenContract.get_balance({
```
Comment (ailisp) [inline environment/commands/transfer-eth-erc20-from-near.js:140]: I feel we'll need some more error recovering. For example, what if burn is success but get errors in following steps? Should we
- revert the burnt after sometime,
- or provide user instructions to continue transfer with a valid burn?

File: README.md (hunk 1 above)
Comment (nearmax) [reply README.md:98]: This is a different key. If we try using the same secret key for different concurrent operations (like relay and transfer) they will conflict through the nonces and one of them will fail.

File: libs-sol/token-locker/contracts/TokenLocker.sol [hunk 4]
```diff
@@ -1,16 +1,116 @@
...
+        }
+        str = string(s);
+    }
+
+    function char(byte b) internal pure returns (byte c) {
+        if (uint8(b) < uint8(10)) return byte(uint8(b) + 0x30);
+        else return byte(uint8(b) + 0x57);
+    }
+
+    function bytes32string(bytes32 b32) internal pure returns (string memory out) {
```
Comment (ailisp) [inline libs-sol/token-locker/contracts/TokenLocker.sol:72]: This, char and uintToString seems to be reused in a few places, move them to NearDecoder or some util.sol?

File: environment/commands/transfer-eth-erc20-from-near.js (hunk 3 above)
Comment (nearmax) [reply environment/commands/transfer-eth-erc20-from-near.js:140]: Yes, we should provide user instructions on how to recover the operation. I haven't added it yet. You could add it too :)

File: rainbow-bridge [hunk 5]
```diff
@@ -0,0 +1 @@
+/Users/maksymzavershynskyi/Projects/rainbow-bridge
```
Comment (ailisp) [inline rainbow-bridge:1]: Added by accident?

File: README.md (hunk 1 above)
Comment (ailisp) [reply README.md:98]: Got it

File: rainbow-bridge (hunk 5 above)
Comment (nearmax) [reply rainbow-bridge]: Fixed.

File: libs-sol/token-locker/contracts/TokenLocker.sol (hunk 4 above)
Comment (nearmax) [reply libs-sol/token-locker/contracts/TokenLocker.sol]: That was debugging code. Removed.

**Comment by nearmax** [comment]
I am going to start merging the current outstanding PRs and resolve the breakages, issues that they cause.

**Pull request merged**